libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/formatter/MapboxDistanceFormatter.kt

```diff
@@ -74,7 +74,7 @@ class MapboxDistanceFormatter(
         val resources = options.applicationContext.resourcesWithLocale(options.locale)
         val unitStringSuffix = getUnitString(resources, smallUnit)
 
-        if (distance <= 0) {
+        if (distance < 0) {
```

Comment:
Can the `distance` be negative? Do we need this check at all? 🤔

Reply:
Going from memory I think the code will crash if the value is below 0.

Reply:
I mean, if that's not an option upstream we may be safely remove the check, aren't we?

Reply:
There's nothing stopping a negative value from being passed into the method. It should protect itself.  There are unit tests to confirm the method's handling of a negative value.